rel/reltool.config.script

```diff
@@ -0,0 +1,24 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+% OTP 26 removed the excl_archive option from reltool.config. However, for <26
+% if we don't specify {excl_archive,..} we end up with small silly *.ez
+% archives instead of ebin directories.
+%
+case erlang:system_info(otp_release) of
+     "26" ->
```

Comment:
Should this also include OTP > 26?

Reply:
Good point. I thought we'd just remove whole .script file at some point. But you're right in 27 it will also fail.

Let's do the opposite and check for 24 and 25 only.

Reply:
I inverted it to append `excl_archive_filters` on 24 and 25 only, otherwise it doesn't do anything. This way it will work for 26, 27, etc.